genisys/modules/ftp.py

```diff
@@ -45,6 +45,8 @@ def generate(self) -> str:
             f"listen_port={ftp_port}",
             f"anon_root={directory}",
             f"listen_address={bind_addr}",
+            "chmod 0644 /first-boot",
+            "chown root /first-boot",
```

Comment:
These needed to be in the preseed, after the ftp command. Sorry if that wasn't clear.